[Bug]: Startup fails when reading ffmpeg version in utils.py

### Please confirm the following before submitting your bug report:

- [x] I searched existing issues for duplicates.
- [x] I have read the README and followed all applicable troubleshooting steps.

### What happened? What did you expect to happen? Is it reproducible every time?

v0.22.0 doesn't show splash screen or main UI. Process starts but then needs to be terminated from System Monitor.
v0.21.0 works fine.

 On Linux Mint, running USDB_Syncer-0.22.0-Linux executable file.



### What OS are you seeing the problem on?

Linux (please specify distribution in the "Additional context" section)

### How do you run the Syncer?

Portable executable

### Relevant log output

```shell
usdb_syncer.log shows:


    2026-06-16 21:05:39 [DEBUG] [init.py:447, MainThread] Qt log message with mode 4: Using Qt multimedia with FFmpeg version 7.1.3 LGPL version 2.1 or later
    2026-06-16 21:05:39 [ERROR] [init.py:273, MainThread] Uncaught exception:
        Traceback (most recent call last):
          File "init.py", line 221, in _run_main
          File "usdb_syncer/gui/mw.py", line 60, in init
          File "usdb_syncer/gui/status_bar.py", line 24, in init
          File "usdb_syncer/gui/status_bar.py", line 52, in set_ffmpeg_status
          File "usdb_syncer/gui/status_bar.py", line 93, in _set_ffmpeg
          File "usdb_syncer/utils.py", line 407, in get_ffmpeg_version
        IndexError: list index out of range



Running the command 'ffmpeg -version' in a terminal gives:


    ffmpeg version 6.1.1-3ubuntu5 Copyright (c) 2000-2023 the FFmpeg developers
    built with gcc 13 (Ubuntu 13.2.0-23ubuntu3)
    configuration: --prefix=/usr --extra-version=3ubuntu5 --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --disable-omx --enable-gnutls --enable-libaom --enable-libass --enable-libbs2b --enable-libcaca --enable-libcdio --enable-libcodec2 --enable-libdav1d --enable-libflite --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libglslang --enable-libgme --enable-libgsm --enable-libharfbuzz --enable-libmp3lame --enable-libmysofa --enable-libopenjpeg --enable-libopenmpt --enable-libopus --enable-librubberband --enable-libshine --enable-libsnappy --enable-libsoxr --enable-libspeex --enable-libtheora --enable-libtwolame --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx265 --enable-libxml2 --enable-libxvid --enable-libzimg --enable-openal --enable-opencl --enable-opengl --disable-sndio --enable-libvpl --disable-libmfx --enable-libdc1394 --enable-libdrm --enable-libiec61883 --enable-chromaprint --enable-frei0r --enable-ladspa --enable-libbluray --enable-libjack --enable-libpulse --enable-librabbitmq --enable-librist --enable-libsrt --enable-libssh --enable-libsvtav1 --enable-libx264 --enable-libzmq --enable-libzvbi --enable-lv2 --enable-sdl2 --enable-libplacebo --enable-librav1e --enable-pocketsphinx --enable-librsvg --enable-libjxl --enable-shared
    libavutil      58. 29.100 / 58. 29.100
    libavcodec     60. 31.102 / 60. 31.102
    libavformat    60. 16.100 / 60. 16.100
    libavdevice    60.  3.100 / 60.  3.100
    libavfilter     9. 12.100 /  9. 12.100
    libswscale      7.  5.100 /  7.  5.100
    libswresample   4. 12.100 /  4. 12.100
    libpostproc    57.  3.100 / 57.  3.100
```

### Additional context


System:
  Kernel: 6.8.0-124-generic arch: x86_64 bits: 64 compiler: gcc v: 13.3.0 clocksource: tsc
  Desktop: Cinnamon v: 6.6.7 tk: GTK v: 3.24.41 wm: Muffin v: 6.6.3 vt: 7 dm: LightDM v: 1.30.0
    Distro: Linux Mint 22.3 Zena base: Ubuntu 24.04 noble

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,13 @@
+<!-- 0.21.1 -->
+
+# Changes
+
+## Features
+
+## Fixes
+
+- ffmpeg version extraction fixed for Linux.
+
 <!-- 0.22.0 -->
 
 # Changes

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,9 +1,5 @@
 <!-- 0.21.1 -->
 
-# Changes
-
-## Features
-
 ## Fixes
 
 - ffmpeg version extraction fixed for Linux.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,7 @@
 <!-- 0.21.1 -->
 
+# Changes
+
 ## Fixes
 
 - ffmpeg version extraction fixed for Linux.

--- a/src/usdb_syncer/utils.py
+++ b/src/usdb_syncer/utils.py
@@ -404,10 +404,10 @@ def get_ffmpeg_version() -> str | None:
         ["ffmpeg", "-version"], capture_output=True, text=True, check=False
     )
 
-    parts = result.stdout.splitlines()[0].split()
-
-    if len(parts) >= 3:
-        return parts[2]
+    output = result.stdout or result.stderr
+    match = re.search(r"ffmpeg version\s+([^\s]+)", output, re.IGNORECASE)
+    if match:
+        return match.group(1)
     return None
 
 

